### PR TITLE
[Enhancement](streamload) stream tvf support user specified label

### PR DESCRIPTION
**Pull request by zzzzzzzs**
## Proposed changes

stream tvf support user specified label
example:

```
curl -v --location-trusted -u root: -H "sql: insert into test.t1 WITH LABEL label1 select c1,c2 from http_stream(\"format\" = \"CSV\", \"column_separator\" = \",\")" -T example.csv http://127.0.0.1:8030/api/_http_stream
```
return:
```
{
    "TxnId": 2064,
    "Label": "label1",
    "Comment": "",
    "TwoPhaseCommit": "false",
    "Status": "Success",
    "Message": "OK",
    "NumberTotalRows": 2,
    "NumberLoadedRows": 2,
    "NumberFilteredRows": 0,
    "NumberUnselectedRows": 0,
    "LoadBytes": 27,
    "LoadTimeMs": 152,
    "BeginTxnTimeMs": 0,
    "StreamLoadPutTimeMs": 83,
    "ReadDataTimeMs": 92,
    "WriteDataTimeMs": 41,
    "CommitAndPublishTimeMs": 24
}
```

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by dataroaring — DISMISSED** [review]
LGTM

**Comment by dataroaring** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/service/FrontendServiceImpl.java [hunk 1]
```diff
@@ -2022,6 +2022,7 @@ private void httpStreamPutImpl(TStreamLoadPutRequest request, TStreamLoadPutResu
             result.getParams().setTableName(parsedStmt.getTbl());
             // The txn_id here is obtained from the NativeInsertStmt
             result.getParams().setTxnConf(new TTxnParams().setTxnId(txn_id));
+            result.getParams().setImportLabel(parsedStmt.getLabel());
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/service/FrontendServiceImpl.java:2025]: update the docs

**Comment by dataroaring** [comment]
run buildall

File: regression-test/suites/load_p0/http_stream/test_http_stream.groovy [hunk 2]
```diff
@@ -600,231 +600,43 @@ suite("test_http_stream", "p0") {
         try_sql "DROP TABLE IF EXISTS ${tableName13}"
     }
 
-    // 14. test parquet orc case
-    def tableName14 = "test_parquet_orc_case"
+    // 14. test label 
+    def tableName14 = "test_http_stream_label"
+    def label = label = UUID.randomUUID().toString().replaceAll("-", "")
```
Comment (mymeiyi) [inline regression-test/suites/load_p0/http_stream/test_http_stream.groovy:605]: def label = UUID.randomUUID().toString().replaceAll("-", "")?

**Comment by mymeiyi** [comment]
It's better to remove label in http header since it does not work
```
    ctx->label = req->header(HTTP_LABEL_KEY);
    if (ctx->label.empty()) {
        ctx->label = generate_uuid_string();
    }
```

File: regression-test/suites/load_p0/http_stream/test_http_stream.groovy (hunk 2 above)
Comment (zzzzzzzs) [reply regression-test/suites/load_p0/http_stream/test_http_stream.groovy:605]: This label is used in SQL and will be compared with the returned results

**Comment by zzzzzzzs** [comment]
> It's better to remove label in http header since it does not work
> 
> ```
>     ctx->label = req->header(HTTP_LABEL_KEY);
>     if (ctx->label.empty()) {
>         ctx->label = generate_uuid_string();
>     }
> ```

I forgot it, removed.

File: regression-test/suites/load_p0/http_stream/test_http_stream.groovy (hunk 2 above)
Comment (mymeiyi) [reply regression-test/suites/load_p0/http_stream/test_http_stream.groovy:605]: Why label is assigned twice:  
`def  label = label = `

**Review by yiguolei — DISMISSED** [review]
LGTM

**Comment by yiguolei** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]
LGTM

**Review by mymeiyi — APPROVED** [review]

**Pull request merged**